tests/contrib/langgraph/test_langgraph_llmobs.py

```diff
@@ -0,0 +1,187 @@
+import json
+
+import pytest
+
+
+def _assert_span_link(from_span_event, to_span_event, from_io, to_io):
+    """
+    Assert that a span link exists between two span events, specifically the correct span ID and from/to specification.
+    """
+    found = False
+    expected_to_span_id = "undefined" if not to_span_event else to_span_event["span_id"]
+    for span_link in from_span_event["span_links"]:
+        if span_link["span_id"] == expected_to_span_id:
+            assert span_link["attributes"] == {"from": from_io, "to": to_io}
+            found = True
+            break
+    assert found
+
+
+@pytest.mark.parametrize(
+    "ddtrace_global_config",
+    [{"_llmobs_enabled": True, "_api_key": "<not-a-real-key>", "_llmobs_ml_app": "unnamed-ml-app"}],
+)
```

Comment:
I know I wrote this but... is there a way we can just write a default fixture to include these config values for all of the below tests and avoid having to write all of the tests in a test class?

Reply:
attempted in https://github.com/DataDog/dd-trace-py/commit/0370e96b7cb19c951a2d3c4bd731ddf0240f982d